include/solo/solo12.hpp

```diff
@@ -341,6 +341,14 @@ class Solo12
         return motor_board_errors_;
     }
 
+    /**
+     * @brief Get pointer to master board interface.
+     */
+    std::shared_ptr<MasterBoardInterface> get_master_board_interface()
+    {
+        return main_board_ptr_;
+    }
+
```

Comment:
I have mixed feelings in exposing the abstraction / main_board_ptr here. The more common way we used in the class is to have getter functions like `get_motor_board_enabled` and could therefore have `get_sent_command_messages` etc.

What do you think?

Reply:
:+1:

Reply:
done